fmriprep/workflows/anatomical.py

```diff
@@ -341,32 +340,46 @@ def inject_skullstripped(subjects_dir, subject_id, skullstripped):
                      output_names=['subjects_dir', 'subject_id']),
         name='skull_strip_extern')
 
-    reconall = pe.Node(
+    autorecon_vol = pe.Node(
+        fs.ReconAll(
+            directive='autorecon2-volonly',
+            openmp=omp_nthreads),
+        name='autorecon_vol')
+    autorecon_vol.interface.num_threads = omp_nthreads
+
+    autorecon_surfs = pe.MapNode(
+        fs.ReconAll(
+            directive='autorecon-hemi',
+            flags=['-noparcstats', '-noparcstats2', '-noparcstats3'],
+            openmp=omp_nthreads),
+        iterfield='hemi',
+        name='autorecon_surfs')
+    autorecon_surfs.interface.num_threads = omp_nthreads
+    autorecon_surfs.inputs.hemi = ['lh', 'rh']
```

Comment:
Yes that is correct. I wonder if this is because MultiProc is not handling MapNodes correctly.